RickLeinecker/Fortuna
# Added Nitro Repair art/functionality

**Description**
Added nitro repair art and functionality. There is a casus variable you can set, and after that you will start using nitro repair. Here's what it looks like:
![NitroRecover](https://user-images.githubusercontent.com/18317476/77205579-8a55b700-6acb-11ea-970a-9bd9c5f820d1.gif)

This currently doesn't restore health because the system for that hasn't been made yet

Resolves #186 

**Type of change**
Check options that are relevant.


**Steps to Verify Functionality**
Make a bulleted/numbered list of steps to verify this feature/bugfix. Include screenshots if necessary.
passes flow.
works as shown in the GIF

**Final Checklist:**
Go down the list and check them off.
